src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -623,9 +624,16 @@ def send_raw_request(cli_ctx, method, uri, headers=None, uri_parameters=None,  #
             logger.warning("Can't derive appropriate Azure AD resource from --url to acquire an access token. "
                            "If access token is required, use --resource to specify the resource")
     try:
-        r = requests.request(method, uri, params=uri_parameters, data=body, headers=headers,
-                             verify=not should_disable_connection_verify())
-        logger.debug("Response Header : %s", r.headers if r else '')
```

Comment:
This is a slight breaking change. Do we need to add `BREAKING CHANGE` to the title to let users notice the change?

Reply:
Not really. Things in `stderr` is never meant to be stable.

Reply:
OK

Reply:
If it is released, please  just remember to inform the customer of this issue.  #12096

Reply:
Sure.